src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -433,7 +433,7 @@ private function readProperty($zval, $property)
      */
     private function getReadAccessInfo($class, $property)
     {
-        $key = $class.'..'.$property;
+        $key = $this->sanitizeAnonymousClassName($class).'..'.$property;
```

Comment:
Thanks for the review@nicolas-grekas. I will implement it like you suggested then.